tcms/testplans/static/testplans/js/get.js

```diff
@@ -2,7 +2,8 @@ var expandedTestCaseIds = [],
     fadeAnimationTime = 500;
 
 const allTestCases = {},
-      autocomplete_cache = {};
+      autocomplete_cache = {},
+      TestCaseStatusConfirmedPK = $('#test_plan_pk').data('testcasestatus-confirmed-pk');
```

Comment:
I think this should be in `.ready()` below to avoid race conditions where the DOM hasn't fully been loaded.

Reply:
I thought about that also, while I tested it worked. I think because we load the JS in the end of the body tag. Also I would like to extract $("#test_plan_pk") to its own variable, to reduce the calls for the permissions also.